helm/cas-provision/values-example.yaml

```
@@ -0,0 +1,11 @@
+# Example of the values needed for this chart
+
+airflowNamespacePrefix: "airflowprefix" # The prefix for the airflow namespace
+ggircsNamespacePrefix: "ggircsprefix" # The prefix for the ggircs namespace
+ciipNamespacePrefix: "ciipprefix" # The prefix for the ciip namespace
```

```suggestion
namespacePrefixes:
  airflow: "airflowprefix"
  ggircs: "ggircsprefix"
  ciip: "ciipprefix"
```

Not sure the comments add any information here